packages/web-react/src/components/FileUploader/FileUploaderAttachment.tsx

```diff
@@ -31,10 +57,13 @@ const FileUploaderAttachment = (props: SpiritFileUploaderAttachmentProps) => {
       {...styleProps}
       className={classNames(classProps.attachment.root, styleProps.className)}
     >
-      <Icon name={iconName} aria-hidden="true" />
-      <span className={classProps.attachment.name}>
-        <span className="text-truncate">{label}</span>
-      </span>
+      {hasImagePreview && imagePreview ? (
+        <AttachmentImagePreview label={label} imagePreview={imagePreview} />
+      ) : (
+        <Icon name={iconName} aria-hidden="true" />
+      )}
+      <span className="text-truncate">{label}</span>
+      {isEditable && <AttachmentActionButton onClick={editHandler}>{editButtonLabel}</AttachmentActionButton>}
```

Comment:
Although it does not make a real difference in this case (yet), please wrap the edit button into a `<span>` with the `FileUploaderAttachment__slot` CSS class. It ensures a uniform appearance across packages and helps us in the future when the slot becomes open to developers.

```suggestion
      {isEditable && (
        <span className={classProps.attachment.slot}>
          <AttachmentActionButton onClick={editHandler}>{editButtonLabel}</AttachmentActionButton>
        </span>  
      )}
```

Reply:
Added span with slot class 👍